pkg/deploy/generator/resources_rp.go

```diff
@@ -548,6 +548,12 @@ yum --enablerepo=rhui-rhel-7-server-rhui-optional-rpms -y install clamav azsec-c
 done
 
 rpm -e $(rpm -qa | grep ^abrt-)
+
+# https://access.redhat.com/security/cve/cve-2020-13401
+cat >/etc/sysctl.d/02-disable-accept-ra.conf <<'EOF'
```

Comment:
There aren't any other configs present by default in these files correct?  Otherwise we'd overwrite them.

Reply:
we are creating the file, it does not exist prior to this pr